docs/continuous-integration/dev-sec-ops/evaluate-oss/README.md

```diff
@@ -0,0 +1,29 @@
+# Evaluate Open Source Software
+
+Identify potential candidates for open-source dependencies and evaluate them against your needs and the required security posture.
+
+## Why Evaluate Open Source Software
+
+Open source software is a critical part of modern software development. It is important to evaluate the open source software uses to ensure it meets the needs and is secure.
+Security is not a given with open source software, and furthermore, what is secure today may not be secure tomorrow so scanning dependencies for known vulnerabilities doesn't always cover all bases.
+This is why we need to look for evidence of a strong security posture and a commitment to security from the maintainers of the open source software we use.
+
+## Applying Open Source Software Evaluation
+
+When evaluating open source software, consider the following:
+
+- Can you avoid adding it as a dependency? The best dependency is the one you don't have.
+- Is it maintained? How often and at what engineering rigor (i.e. code reviews, branch protection, tests)
+- Is there evidence that effort is taken to make it secure?
+- Can you find a reference that it is used significantly downstream by other projects or is referenced by known and trusted documentation?
+- Is it easy to use securely?
+- Are there instructions on how to report vulnerabilities? 
+- Does it have any known vulnerabilities or security issues?
+- Are its dependencies secure, or at least up to date and actively maintained?
+- Has it been audited by a third party such as the [OpenSSF Security Reviews](https://github.com/ossf/security-reviews/blob/main/Overview.md#readme)?
```

Comment:
I would add to check the license of the software, how many stars and forks it has.